jormungandr-lib/src/interfaces/block0_configuration/BLOCKCHAIN_CONFIGURATION_DOCUMENTED_EXAMPLE.yaml

```diff
@@ -207,6 +207,12 @@ initial:
       - address: {initial_funds_address}
         value: 10000
 
+  # Initial token distribution present in the blockchain
+  - token:
+      name: "7e5d6abc"
+      to: {initial_funds_account_identifier}
+      value: 100
```

Comment:
That would hurt readability and bloat the file size. Consider having tens of thousands (that's what we have during funds) of wallets in this form:

```yaml
  - token:
      name: "7e5d6abc"
      to: {initial_funds_account_identifier}
      value: 100
  - token:
      name: "7e5d6abc"
      to: {initial_funds_account_identifier2}
      value: 100
  - token:
      name: "7e5d6abc"
      to: {initial_funds_account_identifier3}
      value: 100
```

Instead you coould follow an example of how are initial fund distributions done but do that per token:

```yaml
initial:
  - token: token_identifier...:
    - address: addr1...
      value: 42
    - address: addr2...
      value: 1337
```

**NB**: Having just a token name is not sufficient considering the general case. Let's go for the agreed human-readable identifier format i.e. `{policy_hash}.{name}`.